main.tf

```diff
@@ -319,3 +321,18 @@ resource "aws_redshift_authentication_profile" "this" {
   authentication_profile_name    = try(each.value.name, each.key)
   authentication_profile_content = jsonencode(each.value.content)
 }
+
+################################################################################
+# CloudWatch Log Group
+################################################################################
+
+resource "aws_cloudwatch_log_group" "this" {
+  for_each = toset([for log in try(var.logging.log_exports, []) : log if var.create && var.create_cloudwatch_log_group])
+
+  name              = "/aws/redshift/cluster/${var.cluster_identifier}/${each.value}"
+  retention_in_days = var.cloudwatch_log_group_retention_in_days
+  kms_key_id        = var.cloudwatch_log_group_kms_key_id
+  skip_destroy      = var.cloudwatch_log_group_skip_destroy
+
+  tags = var.tags
```

Comment:
Please change this to support merge of tags like this: `tags = merge(var.tags, var.cloudwatch_log_group_tags)`